src/wasm-binary.h

```diff
@@ -906,8 +906,13 @@ class WasmBinaryBuilder {
   BinaryConsts::ASTNodes readExpression(Expression*& curr);
   void pushBlockElements(Block* curr, size_t start, size_t end);
   void visitBlock(Block *curr);
-  Expression* getMaybeBlock(WasmType type);
+
+  // Gets a potential list of instructions. This is not a block and cannot be
+  // branched to.
+  Expression* getList(WasmType type);
```

Comment:
This name is confusing. There is no `Expression` that's a list, other than a block. `GetMaybeBlock` makes sense if only because it could return either a block or a single instruction.

Reply:
I guess what this means is that it could return a single instruction or an implicit block (which cannot be branched to, and will not appear in the wasm output)?

Reply:
Yeah, exactly - it's reading a "list context" in a wasm binary, so it could be a list of instructions, but we do know it can't be branched to.

Ideas for a better name? :) Maybe `getExpressionList` or `getExpressionOrList`?